MANIFEST.in

```diff
@@ -1,19 +1,10 @@
-include AUTHORS.rst
-include CONTRIBUTING.rst
-include CHANGELOG.rst
-include LICENSE
-include README.rst
-include kolibri_explore_plugin/VERSION
-include kolibri_explore_plugin/content_types.json
-recursive-exclude kolibri_explore_plugin/* *pyc
-recursive-exclude kolibri_explore_plugin/assets *
-recursive-include kolibri_explore_plugin/migrations *
-recursive-include kolibri_explore_plugin/management *
-recursive-include kolibri_explore_plugin/templates *.*
-recursive-include kolibri_explore_plugin/static *.*
-recursive-exclude kolibri_explore_plugin/static/collections *
-recursive-include kolibri_explore_plugin/build *.json
-recursive-include kolibri_explore_plugin/locale *.mo
-recursive-include kolibri_explore_plugin/locale *.json
-recursive-include kolibri_explore_plugin/vendor *.py
```

Comment:
I wasn't aware of `setuptools_scm`, this is awesome! I haven't looked, but is there anything else that's in git that would now need to be excluded?

Reply:
There’s [these files](https://github.com/endlessm/kolibri-explore-plugin/pull/662#issuecomment-1604393897) and arguably the CI and test files. ttbomk it’s a matter of taste as to whether they go in the sdist: some people argue that the sdist should be usable for development (and hence contain everything from git); others argue that it is only there to build the project from (and hence doesn’t need to include CI stuff, tests, etc.)

Reply:
Yeah, I think you're right. The only times I differ are:

* Something in git that's only used for development and bloats the install.
* Something in git that depends on git history (the `.git` directory) to operate.

I don't think it's worth splitting those hairs here and now.